assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -64,6 +64,12 @@ const fetchCreateAudienceStore = createFetchStore( {
 	},
 } );
 
+const fetchSyncAvailableAudiencesStore = createFetchStore( {
+	baseName: 'syncAvailableAudiences',
+	controlCallback: () =>
+		API.set( 'modules', 'analytics-4', 'sync-audiences' ),
+} );
```

Comment:
Hmm, something that was missed from the AC, but I think we might as well address here, is that we should update the client-side value of the `availableAudiences` setting when handling the response for `syncAvailableAudiences()`.

We should be able to do this easily by following the same approach as `syncAvailableCustomDimensions()`:

https://github.com/google/site-kit-wp/blob/0973df368d41d2b41ba65efd19737c0ce18d1b10/assets/js/modules/analytics-4/datastore/custom-dimensions.js#L83-L92

Reply:
Updated, thanks!